h03_lakesd3.py

```diff
@@ -0,0 +1,101 @@
+#################################################################################
+# Author: Dawson Lakes
+# Username: lakesd3
+#
+# Assignment: HW03
+# Purpose: Draw a Person with colors using functions.
+# Google Doc Link: https://docs.google.com/document/d/1ln0oyXvMbdzL3VVBTqcN4mgtV0iLCc3-Pke13agS4eo/edit?usp=sharing
+#
+#############################################################################
+
+import turtle
+
+
+wn = turtle.Screen()
+wn.bgcolor("blue")
+
+
+stickman = turtle.Turtle()
+stickman.pensize(10)
+
+
+def draw_square_head():
```

Comment:
You probably have the code above outside of main because of scoping issues. The fix is to pass `stickman` into the function when you call it:

```
def draw_square_head(stickman):
   ...

def main():
    stickman = turtle.Turtle()
    draw_square_head(stickman)
```